src/UIAutomation/FunctionalTests/WinRTWrapperTests.cpp

```diff
@@ -0,0 +1,471 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include "pch.h"
+#include "CppUnitTest.h"
+
+#include "ModernApp.h"
+#include "TestUtils.h"
+
+namespace WinRTWrapperTests
```

Comment:
>WinRTWrapperTests [](start = 10, length = 17)

I'm trying to call this the "Builder" library. I think that's a bit better than wrapper, but I'm also open to ideas. :)

Reply:
I also prefer "builder".

Reply:
Sounds good to me.